jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -798,56 +838,6 @@ ecma_builtin_dispatch_construct (ecma_object_t *obj_p, /**< built-in object */
 } /* ecma_builtin_dispatch_construct */
 
 /**
- * Dispatcher of built-in routines
- *
- * @return ecma value
- *         Returned value must be freed with ecma_free_value.
- */
-static ecma_value_t
-ecma_builtin_dispatch_routine (ecma_builtin_id_t builtin_object_id, /**< built-in object' identifier */
```

Comment:
Not necessary, but I would prefer to do this. It is better from logical perspective.